src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/Example5.java

```diff
@@ -5,25 +5,25 @@
       <property name="format" value="^[a-z]([a-z0-9][a-zA-Z0-9]*)?$"/>
       <property name="accessModifiers" value="protected, package, private"/>
       <message key="name.invalidPattern"
-        value="Parameter name ''{0}'' must match pattern ''{1}''"/>
+        value="Name ''{0}'' must match pattern ''{1}''."/>
```

Comment:
please revert this update.
It was done on purpose to show that user can override message
https://github.com/checkstyle/checkstyle/blob/56af76110fd2b6acee4e47db63b7bd4511dc6c15/src/main/resources/com/puppycrawl/tools/checkstyle/checks/naming/messages.properties#L4
https://checkstyle.org/checks/naming/parametername.html#Violation_Messages